src/components/tables/filetreetable.tsx

```diff
@@ -136,7 +136,7 @@ function ByteSizeField(props: TableFieldProps) {
 function PercentBarField(props: TableFieldProps) {
     const now = props.entry.percent ?? 0;
 
-    return <ProgressBar now={now} className="white-outline" />;
+    return <ProgressBar now={now} status={Status.downloading} className="white-outline" />;
```

Comment:
Keep this blue.
Or, if you want to get colorful here too then make skipped files grey, downloading ones blue and finished ones green to be consistent with torrent table.